public/app/features/alerting/unified/RuleList.tsx

```diff
@@ -107,14 +106,14 @@ export const RuleList: FC = () => {
           {promReqeustErrors.map(({ dataSource, error }) => (
             <div key={dataSource.name}>
               Failed to load rules state from{' '}
-              <a href={`${config.appSubUrl ?? ''}/datasources/edit/${dataSource.id}`}>{dataSource.name}</a>:{' '}
+              <a href={locationUtil.assureBaseUrl(`/datasources/edit/${dataSource.id}`)}>{dataSource.name}</a>:{' '}
```

Comment:
do not think you need this, if the url starts without a `/` the base url will be added automatically

Reply:
If you click on it while already inside grafana. But I think the anchor element will still be rendered w/o base url, thus if you copy or click "open in new page" it won't work?

Reply:
nvm there's a `<base />` provided so relative links will work as expected. updating

Reply:
updated to relative